Rename getLogger() to get_logger()

lstein opened the pull request:
## What type of PR is this? (check all applicable)

- [X] Refactor
## Have you discussed this change with the InvokeAI team?

- [ ] Yes
- [X] No, because: trivial fix

      
## Have you updated all relevant documentation?
- [X] Yes
- [ ] No

## Description

It annoyed me that the class method to get the invokeai logger was `InvokeAILogger.getLogger()`. We do not use camelCase anywhere else. So this PR renames the method `get_logger()`.

psychedelicious reviewed (commented):
Why not just totally rename it?

lstein commented:
> Why not just totally rename it?

I don’t generally like to change a published API, but I’ve gone ahead and renamed to `get_logger()` everywhere and removed the old call’s name.

lillekemiker commented:
How did this auto-merge if flake8 didn't pass?
